### PR TITLE
DAF-44 - Make djangosampler Python 3 compatible 

### PR DESCRIPTION
**Pull request by tjphopkins**
To test:

- [ ] pip install djangosampler from this branch:
`pip install git+ssh://git@github.com/conversocial/djangosampler.git@daf-44`
- [ ] visit localhost:8000/sampler/ and check that the page loads without error

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -4,9 +4,10 @@
...
 
 [tox]
-envlist = py27
+envlist = py27, py34
 
 [testenv]
 commands = {envpython} run_tests.py
 deps =
     django==1.8
+    mock==2.0.0
```
Comment (lanzz) [inline tox.ini]: Mock is part of the standard library in Python 3, living under `unittest.mock`. You can do something like
```python
try:
    from unittest.mock import ...
except ImportError:
    from mock import ...
```
And this dependency can be changed to `py27: mock=2.0.0` to limit it to the `py27` tox environment.

File: djangosampler/views.py [hunk 2]
```diff
@@ -1,3 +1,6 @@
+from __future__ import absolute_import
+from __future__ import division
```
Comment (lanzz) [inline djangosampler/views.py]: These can be combined on a single line (opinion only)

File: djangosampler/views.py [hunk 3]
```diff
@@ -34,7 +37,7 @@ def queries(request, query_type, date_string, offset=0, sort='total_duration'):
 
     current_page = 1 + start_offset / PAGE_SIZE
     max_pages = int(ceil(total_queries / float(PAGE_SIZE)))
-    pages = xrange(max(1, current_page - 5), 1 + min(max_pages, current_page + 5))
+    pages = range(max(1, current_page - 5), 1 + min(max_pages, current_page + 5))
```
Comment (lanzz) [inline djangosampler/views.py]: This list (which can probably get pretty big) gets materialized only to be replaced immediately with another list built from a mapping of it. Probably makes sense to keep it a generator on 2.7 (use `six.moves.range`, which maps to `xrange` on 2.7 and to `range` on 3+)

File: djangosampler/test_sampler.py [hunk 4]
```diff
@@ -1,23 +1,85 @@
...
-import sampler
-import models
+from . import sampler
+from . import models
 
-class TestSampler(TestCase):
-    def test_calculate_cost(self):
-        settings.DJANGO_SAMPLER_USE_COST = None
+
+class TestCalculateCost(TestCase):
```
[the remark this answers is not in the document]
Comment (tjphopkins) [reply djangosampler/test_sampler.py]: This exists `test_use_cost_false_returns_zero`, but I'll make it explicit that USE_COST is set to False

**Pull request merged**